tools/audio/sampleconv/src/container/aiff.c

```diff
@@ -0,0 +1,839 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (C) 2024 ZeldaRET
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+#include <assert.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "../util.h"
+#include "../codec/vadpcm.h"
+#include "aiff.h"
+
+typedef struct {
+    int16_t numChannels;
+    uint16_t numFramesH;
+    uint16_t numFramesL;
+    int16_t sampleSize;
+    int16_t sampleRate[5]; // 80-bit float
```

Comment:
Should this be `uint8_t sampleRate[10]`? The two places it's used it's immediately casted